utl/flatbuffer/vw_to_flat.cc

```diff
@@ -452,7 +452,7 @@ void to_flat::convert_txt_to_flat(VW::workspace& all)
     {
       if (!VW::example_is_newline(*ae) ||
           (all.example_parser->lbl_parser.label_type == VW::label_type_t::CB &&
-              !CB_ALGS::example_is_newline_not_header(*ae)) ||
+              !VW::example_is_newline_not_header_cb(*ae)) ||
```

Comment:
might as well rename? do we need the _cb?

Reply:
This one specifically checks the cb label so its not a generally applicable function. This used to be conveyed by the fact it was in the CB_ALGS namespace but without that i figured a suffix keeps that info around